evap/staff/templates/staff_course_copyform.html

```diff
@@ -0,0 +1,44 @@
+{% extends 'staff_course_base.html' %}
+
+{% block content %}
+    {{ block.super }}
+    <h3>
+        {% trans 'Create copy of course' %}
+    </h3>
+    <form id="course-form" method="POST" class="form-horizontal multiselect-form select2form">
+        {% csrf_token %}
+
+        <div class="card mb-3">
+            <div class="card-body">
+                {% include 'bootstrap_form.html' with form=course_form %}
+            </div>
+        </div>
+
+        <div class="card mb-3">
+            <div class="card-body">
+                <h5 class="card-title">{% trans 'Evaluations that will be copied' %}</h5>
+                {% if course.evaluations.count > 0 %}
+                    <ul>
+                        {% for evaluation in course.evaluations.all %}
+                            <li>
+                                {{ evaluation.full_name }}
```

Comment:
I don't think it matters. Do you know how to do that in a template?

Reply:
it would be nice to bring them into an order because we also do it in many other places :)
have a look at, e. g., `contributor/templates/contributor_index.html` and the `dictsort` filter.